.vscode/settings.json

```diff
@@ -4,5 +4,11 @@
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "[xml]": {
     "editor.defaultFormatter": "redhat.vscode-xml"
+  },
```

Comment:
Can you revert the changes here? We want to use prettier if possible.

Reply:
Yup, check it now.  That wasn't a conscious choice, but suggested by vscode while I was working in the IDE.  If you're doing linting stuff on commit/push, maybe the .vscode dir should be ignored?